sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/timeExpressions.scala

```diff
@@ -349,3 +353,109 @@ object SecondExpressionBuilder extends ExpressionBuilder {
   }
 }
 
+/**
+ * Returns the current time at the start of query evaluation.
+ * There is no code generation since this expression should get constant folded by the optimizer.
+ */
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = """
+    _FUNC_([precision]) - Returns the current time at the start of query evaluation.
+    All calls of current_time within the same query return the same value.
+
+    _FUNC_ - Returns the current time at the start of query evaluation.
+  """,
+  arguments = """
+    Arguments:
+      * precision - An optional integer literal in the range [${TimeType.MIN_PRECISION}..${TimeType.MICROS_PRECISION}], indicating how many
```

Comment:
I guess the constants ${TimeType.MIN_PRECISION} won't embedded in the string. It should start from s" at least. Could you generate the doc, and double check.

Reply:
Changed to hardcoded versions in the new revision.